install/update_91_92.php

```diff
@@ -294,7 +294,8 @@ function update91to92() {
               'glpi_devicesoundcardmodels',
               'glpi_devicegenericmodels',
               'glpi_devicebatterymodels',
-              'glpi_devicefirmwaremodels'];
+              'glpi_devicefirmwaremodels',
+              'glpi_devicesensormodels'];
 
```

Comment:
no: this list of tables os intended to create models for several component itemtypes. I re-used it to glpi_ create device_sensormodels.

we were indeed discusssing this with @trasher a few minutes ago